app/src/renderer/components/common/TmPage.vue

```diff
@@ -71,7 +73,7 @@ export default {
       type: String,
       default: ``
     },
-    "menu-body": {
+    "menuBody": {
```

Comment:
```suggestion
    menuBody: {
```

Reply:
I should add a rule for all those, after lunch I'll add this 👍